docs/testcases/UI/System_Management/2232_NAT_Gateway/2232_10.md

```diff
@@ -0,0 +1,28 @@
+# Add several route records with the same server name but different ports
+
+Test verifies that it's possible to add several route records with the same server name but different ports
+
+**Prerequisites**:
+- Admin user
+- Perform [2232_2](2232_2.md) case
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Login as the admin user from the prerequisites | |
+| 2 | Open the **Settings** page | |
+| 3 | Click the **SYSTEM MANAGEMENT** tab | |
+| 4 | Click the **NAT GATEWAY** tab | |
+| 5 | Click **Add Route** button | |
+| 6 | Input `bing.com` into the ***Server name:*** field |  |
+| 7 | Mark ***Specify IP address*** checkbox | |
+| 8 | Input `80` into the ***Port:*** field | |
```

Comment:
Suggest to add a verification here - whether the gateway with the same server name and port already exists. And if yes - try to use another port in this case, for example